### PR TITLE
Grammer refactoring

### PR DESCRIPTION
**Pull request by KyuaKwon**
It was a nogada, but I made it :tada:

Not a big change, but just unifying grammar.

Re-notify: 
There's a issue about setting Timeout that Yangwoo found.
If we use Timeout function for any needed reason, ES6 grammar, especially expressing function as arrow(```=>```), does not work. So I remained few functions as ES5.
It's a public issue with ES6.

@SarahhhC @MechanicKim

### CODE REVIEW

File: src/server/controllers/users.controller.js [hunk 1]
```diff
@@ -278,7 +278,7 @@ export function signIn(req, res, next) {
             });
         } else {
           stampDeviceToken(req.body.deviceToken, existingUser)
-            .then(user => stampUser(user))
+            .then((user) => stampUser(user))
```
Comment (SarahhhC) [inline src/server/controllers/users.controller.js]: Maybe in this case we don't need parenthesis cause it's one line...

Comment (KyuaKwon) [inline src/server/controllers/users.controller.js]: Does it matter?
Was the ES5 version
```
.then(user) {
  return stampUser(user);
}
```
?

Comment (SarahhhC) [inline src/server/controllers/users.controller.js]: @MechanicKim I need your explanation. Please~

Comment (MechanicKim) [inline src/server/controllers/users.controller.js]: Single line function!
As airbnb style guide...

**Unexpected parentheses around single function argument having a body with no curly braces**

user => stampUser(user)

Comment (KyuaKwon) [inline src/server/controllers/users.controller.js]: Thanks a lot!!

**Comment by SarahhhC** [comment]
LGTM!! Nice job @KyuaKwon 🎉 👍

**Pull request merged**